filter/clear_fields.go

```diff
@@ -13,7 +13,7 @@ var ClearFieldsDesc = baker.FilterDesc{
 	Name:   "ClearFields",
 	New:    NewClearFields,
 	Config: &ClearFieldsConfig{},
-	Help:   "Clear a set of fields of all records passing through",
+	Help:   "Clear a set of fields (makes them empty) of all records passing through",
```

Comment:
If clear is not so clear (no pun intended :-) ) maybe we can just use:
```suggestion
	Help:   "Reset a set of fields of all records passing through",
```